.github/workflows/nightly-test-cpu.yml

```
@@ -25,13 +25,10 @@ jobs:
     - name: Install
       run: |
         pip install -U pip wheel
-        pip install --pre torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/nightly/cpu
-        pip install pytest
-        pip install matplotlib tensorboard ipython ipywidgets pandas optuna onnx onnxruntime pytorch-ignite
-        pip install -v -e .
+        pip install -v -e . --pre torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/nightly/cpu 
```

I guess this is needed?

```suggestion
        pip install -v -e .[test] --pre torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/nightly/cpu 
```

---

Fixed it!